staging/src/k8s.io/client-go/tools/cache/delta_fifo.go

```diff
@@ -146,7 +146,7 @@ const (
 	// as well. Hence, Replaced is only emitted when the option
 	// EmitDeltaTypeReplaced is true.
 	Replaced DeltaType = "Replaced"
-	// Sync is for synthetic events during a periodic resync.
+	// Sync is for synchronous events during a periodic resync.
```

Comment:
"synthetic" was correct, drop this change

Reply:
Got it, fixed. PTAL thanks.